### ISSUE TITLE
Subtitles don't move to the visible portion of the video in vo=gpu-next

### ISSUE THREAD

**Issue by 1Naim** [mpv-player/mpv#9928]
### Important Information

Provide following Information:
- mpv version: https://github.com/mpv-player/mpv/commit/b1fb4b783bd604ffb763e88a7cee2ad9d9c2c209
- Windows Version: Windows 10 21H1 19043.1526
- Source of the mpv binary: shinchiro's build
- If known which version of mpv introduced the problem: probably #9870 
- Possible screenshot or video of visual glitches

If you're not using git master or the latest release, update.
Releases are listed here: https://github.com/mpv-player/mpv/releases

### Reproduction steps

1.  Play a non 16:9 aspect video (in this case it's 1920x816)
2.  Take a screenshot

### Expected behavior
![image](https://user-images.githubusercontent.com/96768389/155955505-4c27518a-1b9d-4387-bac9-505d292218f7.png)
Subtitles are moved to the visible portion of the video (taken with screenshot window in vo=gpu and blend-subtitles=yes)
### Actual behavior
![image](https://user-images.githubusercontent.com/96768389/155955694-6ac34d6a-ecaf-4525-aa47-b590171f1fc3.png)
Subtitles are not there
![image](http://midi.moe/f/vafdk.png)
This is the image taken with screenshot window in vo=gpu-next. Subtitles are shown on the the black bars but are not visible during playback
### Log file
http://0x0.st/oK78.txt

**Comment by yeezylife** [comment]
![ANISAB-BD  Memories Segment 1 Magnetic Rose  BD-rip 1920x1056 x264 FLAC - 00_17_38](https://user-images.githubusercontent.com/30175425/155914995-1dd3142b-b2bb-4aae-a670-5d76e1d7a1b4.jpg)

![无标题](https://user-images.githubusercontent.com/30175425/155918801-f9cdc4d4-f1ce-4b70-8e35-35d87528a90b.jpg)

Reproduced (both).

**Comment by 1Naim** [comment]
Subtitle appears in the blackbars if video-sync isn't display-resample (haven't tested all the options, but commenting video-sync=display-resample) makes it appear for reasons unknown to me. Thought I should mention this since the other issue got closed for not following the issue template

**Comment by Obegg** [comment]
Ok, so I experience this issue too, and here are my findings:

I found a YouTube video that has this issue:
https://www.youtube.com/watch?v=boKmZKTKXHc
Watching on a 1080p screen, mpv on fullscreen - subtitles are cut on black borders:
![Screenshot 2022-03-05 204608](https://user-images.githubusercontent.com/13978119/156896285-3ff01518-bd9c-4f7c-ad9a-12c65a28a44a.jpg)
And trying to screenshot from inside mpv just moves the subtitles to a different location and different size.

**Comment by yeezylife** [comment]
And I don't think `screenshot video` should show subtitles at all.Now I have to turn subtitles on and off constantly...

**Comment by mnisius** [comment]
This Bug is super annoying hope it will be fixed soon... 

Is there any other workaround without disabling  vo=gpu-next or video-sync=display-resample as both options are important to me...

**Comment by yeezylife** [comment]
> That's the difference of `s` and `S`, the former keeps subtitles and the latter excludes them. 

Both `s` and `S` keeps subtitles now(in the latest build with gpu-next),that's why I reported it.

**Comment by snylonue** [comment]
same issue with an srt substitle

**Comment by ekce** [comment]
I'm still getting this bug with the current git (using arch linux and mpv-git from the AUR).

**Comment by Obegg** [comment]
Can confirm issue still exists, I've noticed if I pause and resume playback the subtitles are shown correctly, but for a split second, so its not a solution.

**Comment by mnisius** [comment]
Yes this still exists. Here is the open issue: https://github.com/mpv-player/mpv/issues/10272
